wfexs_backend/fetchers/git.py

```diff
@@ -388,17 +389,29 @@ def fetch(
 HEAD_LABEL = b"HEAD"
 REFS_HEADS_PREFIX = b"refs/heads/"
 REFS_TAGS_PREFIX = b"refs/tags/"
+GIT_SCHEMES = ["https", "git+https", "ssh", "git+ssh", "file", "git+file"]
```

Comment:
I believe these should be limited to the `git+` forms only? As pure URLs of the schemes should not be handled by the git fetcher

Reply:
I agree